src/NUnitFramework/framework/Properties/AssemblyInfo.cs

```diff
@@ -40,7 +40,7 @@
 [assembly: AssemblyTitle("NUnit Framework .NET 4.0")]
 #elif NET_2_0
 [assembly: AssemblyTitle("NUnit Framework .NET 2.0")]
-#elif PORTABLE
+#elif NETSTANDARD1_3
 [assembly: AssemblyTitle("NUnit Framework Portable")]
```

Comment:
Stray "Portable" here. Although looks like this file is generally out of date, it's missing the .NET 3.5 and .NET Standard 1.6 builds too.